test.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
+const { ESLint } = require("eslint");
+
+describe("validate config", () => {
```

Comment:
We should have this test in all of our config repos!

Reply:
I also wonder if these repo's could use a more up-to-date version of `node` to use the built-in test runner to avoid the `jest` dependency 🤔

Reply:
This is very cool!
Was wondering some time ago how to validate the eslint config against the repo itself (e.g. sorting keys alphabetically). This seems to be a good foundation to achieve that.

Reply:
In the past we've handled bumping node in all projects as a standalone project. As the number of projects grow, I believe we're going to need to put together a better strategy to get incremental adoption, but it's definitely something we'll need to tackle sooner rather than later